source/develop/dev-process/working-with-gerrit.md

```diff
@@ -107,7 +111,7 @@ Or alternatively without need of authentication:
 
       wget -P .git/hooks https://gerrit.ovirt.org/tools/hooks/commit-msg
 
-**ATTENTION**: All of the ovirt projects require a Signed-off-by (SOB), add the following lines in the end of .git/hooks/commit-msg, after the line "add_ChangeId":
+**ATTENTION**: All of the ovirt projects require a Signed-off-by (SOB), add the following lines in the end of `<ovirt_engine_repo_location>/.git/hooks/commit-msg`
```

Comment:
This instruction is not only for ovirt engine repo, it can be also for other repos of ovirt.gerrit. I wouldn't change it.

Reply:
Ok, I will fix